middlewares/authentication.js

```diff
@@ -0,0 +1,24 @@
+import jwt from 'jsonwebtoken';
+import dotenv from 'dotenv';
+
+dotenv.config();
+
+const secret = process.env.SECRET_KEY;
+
+const time = { expiresIn: '72hrs' };
+const generateToken = payload => jwt.sign({ payload }, secret, time);
+
+const verifyToken = (req, res, next) => {
+  const token = req.headers.accesstoken;
+  jwt.verify(token, secret, (error, decoded) => {
+    if (error) {
+      error = new Error('Invalid authencation! Can you check and try again?');
```

Comment:
There's a namespace clash between `error` on line 13/14 and the one on line 15. You could rename any one of them to `err`. Doesn't really matter which one. Still works.